pandas/core/groupby/ops.py

```diff
@@ -464,7 +464,7 @@ def _cython_operation(self, kind, values, how, axis, min_count=-1, **kwargs):
         # are not setup for dim transforming
         if is_categorical_dtype(values) or is_sparse(values):
             raise NotImplementedError(
-                "{} are not support in cython ops".format(values.dtype)
+                "{} are not supported in Cython operations".format(values.dtype)
```

Comment:
Would this be clearer as "{dtype} dtype is not supported [...]"?

Reply:
I think so, feels like error messages shouldn't expose implementation details like this

Reply:
yah, the groupby code involves a lot of nested try/except catching, so i dont know off the top of my head whether this gets expose to the user.

Regardless, small improvements are still improvements, and they're easy to review.